lib/mllp/sender.ex

```diff
@@ -343,6 +314,48 @@ defmodule MLLP.Sender do
     stop_connection(state, reason, "process terminated")
   end
 
+  @should_reconnect_errors [
```

Comment:
I think this is about all we can unequivocally handle as far as saying  "Ok, we definitely need to try to reconnect". `:econnrefused` is an odd one, but I suppose that can happen when sending or  receiving. I think there might be other cases, but I'm reluctant to add them unless I'm 100% sure it should result in a reconnect.

Reply:
`econnrefused` is fine, I have seen it most when receiving port is not open